packages/core/src/Stores/client-store.js

```diff
@@ -727,6 +729,10 @@ export default class ClientStore extends BaseStore {
         return this.account_status?.status?.includes('dxtrade_password_not_set');
     }
 
+    get is_authenticated_with_idv_photoid() {
```

Comment:
@likhith-deriv Why not a hook?

Reply:
@likhith-deriv I still have the same question 👀 
Please let's not add more code to the legacy stores 🙏🏻